lib/generators/logidze/install/templates/migration.rb.erb

```diff
@@ -16,17 +16,26 @@ class <%= @migration_class_name %> < ActiveRecord::Migration
     end
 
     execute <<-SQL
-      CREATE OR REPLACE FUNCTION logidze_version(v bigint, data jsonb) RETURNS jsonb AS $body$
+      CREATE OR REPLACE FUNCTION logidze_version(v bigint, data jsonb, blacklist text DEFAULT '', whitelist text DEFAULT '') RETURNS jsonb AS $body$
```

Comment:
Maybe we can use here `blacklist text[]` to avoid `string_to_array` for every operation?